crates/vm/levm/src/utils.rs

```diff
@@ -60,7 +60,7 @@ pub fn new_vm_with_ops_addr_bal(bytecode: Bytes, address: Address, balance: U256
         address,
         Default::default(),
         Default::default(),
-        U256::MAX,
+        U256::from(30000000), // arbitrary gas limit for now...
```

Comment:
Is there any particular reason for changing this?

Reply:
Yes, because it breaks tests and it was a quick fix. I now figured out why. 
We are saying that a callframe starts with 0 gas consumed but we are not taking into account the Transaction Base Cost. So we were letting it consume more gas than it should have been capable of. The other quick fix that I implemented now is just initializing the first callframe with an initial `gas_used = TX_BASE_COST`. Now it doesn't break tests :)

Reply:
Yes that sounds nice, but I was asking about another thing. Here we were setting `gas_limit`  to be `U256::MAX` just for testing purposes. Anyways, probably `U256::from(30000000)` is ok too.